sdk/python/feast/feature_store.py

```diff
@@ -34,7 +34,7 @@
 from feast.protos.feast.types.EntityKey_pb2 import EntityKey as EntityKeyProto
 from feast.registry import Registry
 from feast.repo_config import RepoConfig, load_repo_config
-from feast.telemetry import Telemetry
+from feast.telemetry import Telemetry, public_method
```

Comment:
Do we still need a `Telemetry` object if we are using `public_method`?

Reply:
It can almost but not entirely be rolled in - the problem is for `get_online_features` the `_telemetry_counter` needs a long-standing Telemetry() object either on the FeatureStore class or elsewhere. One way around it would be to have a global Telemetry() instance which I think we had at one point, wdyt?

Reply:
Yea, I think the [Global Object Pattern](https://python-patterns.guide/python/module-globals/) may make sense here.

Reply:
wdyt?